content/en/docs/reference/kubectl/conventions.md

```diff
@@ -22,9 +22,10 @@ For a stable output in a script:
 
 ## Subresources
 
-* You can use the `--subresource` alpha flag for kubectl commands like `get`, `patch`,
+* You can use the `--subresource` beta flag for kubectl commands like `get`, `patch`,
 `edit` and `replace` to fetch and update subresources for all resources that
 support them. Currently, only the `status` and `scale` subresources are supported.
+  * Please note that the `scale` subresource is not supported for use with the `--subresource` flag for the `kubectl edit` command.
```

Comment:
(nit)
Try something like this, where we explain what happens:
```markdown
  * For `kubectl edit`, the `scale` subresource is not supported. If you use  `--subresource` with
    `kubectl edit` and specify `scale` as a subresource, DEMONS FLY OUT OF YOUR NOSE.
```
(update **DEMONS FLY OUT OF YOUR NOSE** to match actual outcome)